backend/migrator/migration/prod/2.23/0000##principal_profile.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE principal ADD profile JSONB NOT NULL DEFAULT '{}';
```

Comment:
How about the `payload`/`config`, so that we can save `profile` as `payload`.`profile`

```suggestion
ALTER TABLE principal ADD profile JSONB NOT NULL DEFAULT '{}';
```

Reply:
If it's `config`, I prefer to move the `mfa_config` into the `config`. But I'm not sure if we need to go so far away right now.

Reply:
I think we can leave `mfa_config` for later. The main point is that it will be more easy to add things in the future.

Reply:
Seems we can keep the `profile`, and move the `mfa_config` into it?